### ISSUE TITLE
Support NDIR sensor

### ISSUE THREAD

**Issue by generateui** [Schluggi/AIOsense#232]
Thanks for this awesome project!

The BME680 is nice, but does not seem to work very well for CO2 sensing: https://dm1cr.de/co2-sensor-comparison-scd30-mh-z19b-bme680-vz89te-sgp30. More of these reports can be found if you dig deeper. It seems that in order to emasure CO2 properly, an NDIR sensor is the gold standard.

**Comment by Schluggi** [comment]
That's correct. The BME680 does not measure CO² at all. It just assume the value based on the the other values. That's why it's called `CO² equivalent` and not `CO²`.

Real CO² sensors are really big because they need to heat the air for measurement. Therefore we diced not to support one.

However you can use I²C to connect the SCD30 anyway. It just don't fit into the case. Feel free to suggest an PR for supporting the sensor on software site.
